bokeh/document/document.py

```diff
@@ -861,6 +862,8 @@ def set_select(self, selector, updates):
             None
 
         '''
+        if isclass(selector) and issubclass(selector, Document):
```

Comment:
need to check for `Model` subclasses in both places.

Reply:
Oh allright. Only `Model` in both cases. Got it.